libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -634,3 +634,7 @@ userdata uint32_t manual_operator __tostring uint32_t___tostring
 userdata uint32_t manual toint uint32_t_toint 0
 userdata uint32_t manual tofloat uint32_t_tofloat 0
 
+include AP_EFI/AP_EFI_MAV.h
+
+singleton AP_EFI_MAV rename efi_mav
+singleton AP_EFI_MAV method get_EFI_state float uint8_t 0 16
```

Comment:
Scripting has EFI bindings and understands the EFI state structure. We would just have to change it to `read write` here:
https://github.com/ArduPilot/ardupilot/blob/3c101d879ee89e4d572d75403a60b038988b7e71/libraries/AP_Scripting/generator/description/bindings.desc#L578-L597

and here:

https://github.com/ArduPilot/ardupilot/blob/3c101d879ee89e4d572d75403a60b038988b7e71/libraries/AP_Scripting/generator/description/bindings.desc#L571-L575

Then we would just need a binding for `AP_EFI::get_state`.